packages/lisk-transactions/src/transactions/base.ts

```diff
@@ -61,7 +72,8 @@ export abstract class BaseTransaction {
 	public readonly type: number;
 	public readonly asset: TransactionAsset = {};
 	public readonly receivedAt: Date = new Date();
-	public readonly isMultisignature?: boolean = false;
+	public readonly containsUniqueData?: boolean;
```

Comment:
why did this become property instead of abstract func?

Reply:
discussed with @SargeKhan briefly and decided for all types we know indefinitely if it contains unique data or not based on `type`, therefore no need for func. thoughts?